src/app/style/utils/normalize.scss

```
@@ -0,0 +1,36 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+body,
```

Не уверен, умеет ли `stylelint` следить за этим, но давай возьмём за правило, что между любыми двумя многострочными блоками кода должен быть отступ в одну пустую строку.

```suggestion
}

body,
```

---

да  ```stylelint``` это исправит